sdk/include/platform/ibex/platform-hardware_revoker.hh

```diff
@@ -13,6 +15,10 @@
 #	error Memory map was not configured with a revoker device
 #endif
 
+DECLARE_AND_DEFINE_INTERRUPT_CAPABILITY(revokerInterruptCapability,
+                                        RevokerInterrupt,
```

Comment:
We expect this to come in from the `board.json`?

Reply:
Yup.  The number in the JSON must match the one that the scheduler raises.